src/Juno/Consensus/Pure/Handle/AppendEntriesResponse.hs

```diff
@@ -152,3 +152,10 @@ processSetAer ks s = go [] (Set.toDescList s)
       | otherwise = case aerReVerify ks aer of
                       Left f -> go (f:fails) rest
                       Right () -> (Just $ aer {_aerWasVerified = True}, fails)
+
+
+
+aerReVerify :: KeySet -> AppendEntriesResponse -> Either String ()
+aerReVerify  _ (AppendEntriesResponse _ _ _ _ _ _ True _) = Right ()
```

Comment:
Maybe use record pattern matching on the specific  fields instead of _ s:
`(AppendEntriesResponse { _aerWasVerified=True })
 (AppendEntriesResponse { _aerProvenance=NewMsg })
`